pkg/cmd/cli/cmd/logs.go

```diff
@@ -163,6 +168,13 @@ func (o *OpenShiftLogsOptions) RunLog() error {
 		}
 		return o.runLogsForBuild(build)
 
+	case "dc", "deploymentconfig", "deploymentconfigs":
```

Comment:
this is spreading, which isn't good. normalization, lowercasing, pluralization, and alias->actual should happen in one place. https://github.com/openshift/origin/pull/4947/files#diff-8040eed03335aa79518a88a4de565decR37 looks promising. Add a TODO to replace the lowercase above with a call to a single helper function, and clear out the singular and alias checks in this function